selfservice/hook/web_hook_integration_test.go

```diff
@@ -934,8 +936,8 @@ func TestWebHooks(t *testing.T) {
 		// error.
 
 		var wg sync.WaitGroup
-		wg.Add(3) // HTTP client does 3 attempts
```

Comment:
that's no longer true assuming this https://github.com/Wikia/kratos/pull/97/files#r1479563845